### ISSUE TITLE
Exoplayer - unable to display RTSP url (Black screen)

### ISSUE THREAD

**Issue by QuentinCln** [TheWidlarzGroup/react-native-video#1069]
### Current behavior
unable to display video (Black screen) when uri source is set to an RSTP uri
Same behavior with Android MediaPlayer.

react-native: 0.55.4
react-native-video: 2.1.1

### Reproduction steps
Also tried to pass ```const isNetwork = !!(uri && uri.match(/^https?:/));``` to true, but no effect

```
        <Video
            source={{
              uri: "rtsp://184.72.239.149/vod/mp4:BigBuckBunny_175k.mov"
              // uri: "rtsp://95.253.236.60/live/ch00_0"
            }}
            rate={1.0}
            volume={1.0}
            muted={false}
            paused={false}
            resizeMode="cover"
            repeat={true}
            onLoadStart={() => {
              console.log("onLoadStart");
            }}
            onLoad={() => {
              console.log("onLoad");
            }}
            onProgress={() => {
              console.log("onProgress");
            }}
            onEnd={() => {
              console.log("onEnd");
            }}
            onError={() => console.log("onError")}
            onBuffer={() => console.log("isBuffering")}
            style={{ width: Dimensions.get("window").width, height: 300 }}
          />
```

### Platform
Which player are you experiencing the problem on:
Ubuntu, physical Android  device
Android ExoPlayer

### Video sample
rtsp://184.72.239.149/vod/mp4:BigBuckBunny_175k.mov
rtsp://95.253.236.60/live/ch00_0

**Comment by shaoting0730** [comment]
http://wxsnsdy.tc.qq.com/105/20210/snsdyvideodownload?filekey=30280201010421301f0201690402534804102ca905ce620b1241b726bc41dcff44e00204012882540400&bizid=1023&hy=SH&fileparam=302c020101042530230204136ffd93020457e3c4ff02024ef202031e8d7f02030f42400204045a320a0201000400    
won't play

**Comment by cobarx** [comment]
@QuentinCln Per https://github.com/google/ExoPlayer/issues/55, RTSP isn't supported by ExoPlayer yet, however there is a PR open to add support in https://github.com/google/ExoPlayer/pull/3854. Once that lands and ExoPlayer puts out a new release that includes that PR, we should be able to get support automatically. I'll keep an eye on it and update. Clearly we need to document which formats are supported on which players, I'll add that to my list of things to do.

@pheromone That URL isn't a RTSP URL. Please open a separate issue for this and add info about what kind of error you are getting. That URL is working fine for me.

**Comment by QuentinCln** [comment]
Thanks @cobarx for the informations. I finally made a wrapper around VideoView that manages RTSP 
meanwhile

**Comment by joshuapinter** [comment]
@QuentinCln Checking in 8 months latest and ExoPlayer still hasn't merged in that PR for RTSP support. 

Any chance you can elaborate on your `VideoView` wrapper solution? 

Thanks!
